### PR TITLE
More outline fixes

### PR DESCRIPTION
**Pull request by pavpanchekha**
This PR fixes the subtler outline issues identified in #1219.

### CODE REVIEW

File: src/lab15.py [hunk 1]
```diff
@@ -283,7 +283,7 @@ def add_inline_child(self, node, w, child_class, frame, word=None):
         line = self.children[-1]
         previous_word = line.children[-1] if line.children else None
         if word:
-            child = child_class(node, line, previous_word, word)
+            child = child_class(node, word, line, previous_word)
```
Comment (chrishtr) [inline src/lab15.py:286]: Was this a real bug in the code?

Comment (pavpanchekha) [reply src/lab15.py:286]: Ah, no, ok, here's what's going on. Between Ch 14 and 15, we change where the `word` argument goes in the argument order. In 14 and prior, the `word` argument goes after `node`; in 15 and 16, it goes at the end. This came about due to various refactors, but isn't necessary now, so I'm fixing it so the argument order doesn't change for no reason.

File: book/scripts.md [hunk 2]
```diff
@@ -309,14 +309,12 @@ ignore the crash. Web pages shouldn't be able to crash our browser!
 You can implement that like this:
 
 ``` {.python}
-class Tab:
```
Comment (chrishtr) [inline book/scripts.md:312]: yikes

File: src/lab9.py [hunk 3]
```diff
@@ -45,8 +45,11 @@ def __init__(self, tab):
         self.node_to_handle = {}
         self.handle_to_node = {}
 
-    def run(self, code):
-        return self.interp.evaljs(code)
+    def run(self, script, code):
```
Comment (chrishtr) [inline src/lab9.py:48]: why is this change necessary?

File: book/scripts.md (hunk 2 above)
Comment (pavpanchekha) [reply book/scripts.md:312]: Yeah, this was kinda ugly. Basically, in Chapter 9 we put the error-handling code in `Tab.load`; in Chapter 12 we want to move it to `run` because then we can use `Task(self.js.run, ...)`. Better put it in its final location from the beginning.

File: src/lab9.py (hunk 3 above)
Comment (pavpanchekha) [reply src/lab9.py:48]: See above comment.

File: book/scripts.md (hunk 2 above)
Comment (chrishtr) [reply book/scripts.md:312]: Is this an orthogonal fix you found along the way? Or does it benefit the outline in a concrete way?

Comment (pavpanchekha) [reply book/scripts.md:312]: It benefits the outline because moving the error handling into `JSContext.run` requires adding a `script` parameter, meaning that the outline changes (a new parameter appears).
